php_phongo.c

```diff
@@ -2445,6 +2445,12 @@ PHP_MINIT_FUNCTION(mongodb)
 
 	/* Initialize libmongoc */
 	mongoc_init();
+
+#ifdef MONGOC_EXPERIMENTAL_FEATURES
```

Comment:
> If you want to skip this release, then that is completely up to you. It might proof problematic for distros as there is no libmongoc-priv anymore, so you'd have to change the requirements to max 1.3. Other then then, I can't think of anything bad happening.

I imagine that would be a problem the moment libmongoc 1.4.0 lands in a distro, unless they had a new version of PHPC to publish at the same time.

/cc @oerdnj @remicollet